tonie_podcast_sync/toniepodcastsync.py

```diff
@@ -13,12 +13,9 @@
 from rich.progress import track
 from rich.table import Table
 from tonie_api.api import TonieAPI
+from tonie_api.models import CreativeTonie
 
-from podcast import (
-    Episode,
-    EpisodeSorting,  # noqa: F401
-    Podcast,
-)
+from tonie_podcast_sync.podcast import Episode, Podcast
```

Comment:
I faced some issues with my existing scripts because it could not import EpisodeSorting in my scripts. My previous workaround was like:

```python
from tonie_podcast_sync.podcast import (
    Episode,
    EpisodeSorting,  # noqa: F401
    Podcast,
)
```

which resolved that. Do you think we could go with that as well in this case? (would help users with existing scripts)